templates/nginx-conf.j2

```diff
@@ -47,6 +47,12 @@ http {
     sendfile        on;
     #tcp_nopush     on;
 
+    {% if nginx_proxy_conf_http is defined %}
+      {% for http_conf_line in nginx_proxy_conf_http %}
+{{ http_conf_line }}
```

Comment:
Indent this by 4 spaces to match the rest of the file. Also can you add `;` at the end (and remove it from the README example) to match the style in the the other templates.